### DISCUSSION TITLE
What is the correct way to provide parameters to a parameterised minting policy written with Aiken?

### DISCUSSION THREAD

**Discussion by dchambers** [MeshJS/mesh#234] [Q&A] [answered]
The only examples of this I can find within the codebase are using `@meshsdk/mesh-csl`, but this library seems to very new, plus is giving me a cryptic WASM call stack (Rust compiled to run on WASM?) and so I'm wondering if there's a different API that I should be using for this?

Thanks in advance for any help!

**Comment by jinglescode** [comment, marked as answer]
> The only examples of this I can find within the codebase are using `@meshsdk/mesh-csl`, but this library seems to very new, plus is giving me a cryptic WASM call stack (Rust compiled to run on WASM?) and so I'm wondering if there's a different API that I should be using for this?
> 
> 
> 
> Thanks in advance for any help!

See https://meshjs.dev/apis/transaction/smart-contract#applyParamToScript
